sites/website/src/docs/resources/cheat-sheet.md

```diff
@@ -0,0 +1,727 @@
+```
+
+---
+
+### [@microsoft/fast-foundation](https://www.fast.design/docs/introduction)
+
+**A library that provides foundational building blocks that can be assembled to create new design systems and component libraries.**
+
+* Exports of this package can generally be thought of as un-styled base components that implement semantic and accessible markup and behavior.   
+* Use this library when you want to create something like Google's Material Design or Bootstrap.
+
+**Example:**
+
+```ts
+import {
+  Button as FoundationButton,
+  buttonTemplate as template,
+} from "@microsoft/fast-foundation";
+import { css } from "@microsoft/fast-element";
+
+export class MyButton extends FoundationButton {
+  ...
+}
+
+export const buttonStyles = css`
+  :host {
+    background-color: azure;
+  }
+`;
+
+export const myButton = MyButton.compose({
+  baseName: "button",
+  baseClass: FoundationButton,
+  template,
+  styles: buttonStyles,
+  shadowOptions: {
+    delegatesFocus: true,
+  },
+});
+```
+
+---
+
+### [@microsoft/fast-components](https://www.fast.design/docs/components/getting-started)     
+
+**The FAST component library and visual system [FAST Frame](https://www.fast.design/docs/design-systems/fast-frame/), which implements fast-foundation.**
+
+* Assembles the building blocks of `@microsoft/fast-foundation` to create its component set. 
+* Use this library when you want to integrate [FAST Components](#using-components) into an existing site or app.
+
+To install the components, use either `npm` or `yarn`:
+
+```shell
+npm install --save @microsoft/fast-components
+``` 
+
+```shell
+yarn add @microsoft/fast-components
+```
+
+To use a component as a custom element in HTML, the custom element must be registered.
+
+**Example:**
+
+```ts
+import { 
+  fastButton, 
+  provideFASTDesignSystem 
+} from "@microsoft/fast-components";
+
+provideFASTDesignSystem()
+  .register(
+    fastButton() // custom element registration
+  );
+```
+
+---
+### [@fluentui/web-components](https://docs.microsoft.com/en-us/fluent-ui/web-components/)
+
+**A component library that implements Microsoft's [Fluent Design System](https://www.microsoft.com/design/fluent/#/).**
+
+* Assembles the building blocks of `@microsoft/fast-foundation` to create its component set. 
+* Use this library when you want components to look and feel like those found in Microsoft products.
+
+To install the components, use either `npm` or `yarn`:
+
+```shell
+npm install --save @fluentui/web-components
+``` 
+
+```shell
+yarn add @fluentui/web-components
+```
+
+To use a Web Component as a custom element in HTML, the custom element must be registered.
+
+**Example:**
+
+```ts
+import { 
+  fluentButton, 
+  provideFluentDesignSystem 
+} from "@fluentui/web-components";
+
+provideFluentDesignSystem()
+  .register(
+    fluentButton() // custom element registration
+  );
+```
+
+Learn more about [Fluent UI Web](https://github.com/microsoft/fluentui).
+
+---
```

Comment:
@huynhicode Sorry to nitpick - I don't think we should have anything regarding Fluent in the cheat sheet. Fluent is an implementation of FAST and not related directly outside our involvement. If we can pull any references to Fluent that will likely save us some headache in the long run. This documentation all exists in multiple places on the Fluent side, so I don't think there's anything lost by removing this from here.

Reply:
@chrisdholt Understandable - I will remove, thank you!

Reply:
While we don't need a full code example here, I think there should be some link over to Fluent as another design system that our team builds. Once we revamp our home page, I think we could remove it, if we add a design systems section to our home page to showcase the systems built on top of FAST. But I think having some sort of link here even for SEO is important. And our community often moves back and forth between FAST and Fluent content.

Reply:
Is the cheat sheet the right place for that though? We have references to Fluent on the docs as it currently stands and it's duplicative content that seems mostly incidentally related because of our relationship to the project. While our Microsoft team moves back and forth, if the cheat sheet is primarily for the community I think it'll be more confusing than not to tie the two directly together. While I get the value to onboarding someone on our direct team, I think the broader story for the community isn't as clear.

Would comments help here in terms of what things actually are? Or perhaps we can mention that replacing FAST with Fluent in most all cases will work for that repo? I'm just very hesitant to link the two when from a community standpoint they are only related in that one (Fluent) is an implementation of the other.

Reply:
@EisenbergEffect @chrisdholt Sorry, I didn't see this until too late.  🤦🏻‍♀️ 😞